### PR TITLE
Cognito User Pool Backup and Restore

### PR DESCRIPTION
**Pull request by bryan-robitaille**
# Summary | Résumé

Creates two utility scripts that can backup user information (excluding passwords) and restores user information to a new User Pool.

## Backup command

`yarn backup {user_pool_id} {csv_file_name}`

- `{user_pool_id}` is the id of the user pool you want to retrieve users from.
- `{csv_file_name}` is the name of the csv file that will be created with the user information.
- Creates a file named `user_pool_{user_pool_id}_users.csv`

## Restore command

`yarn restore {user_pool_id}`

- `{user_pool_id}` is the id of the user pool you want to restore users to.

## User Flow
After running the `restore` command the next time a user attempts to login they will be redirected to the Forgot Password page.  Resetting their password through the application will provide them access to their account and previous saved forms.

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [x] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [x] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [x] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [x] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [x] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [ ] Have you modified the change log and updated any relevant documentation?
- [ ] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [ ] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

File: utils/cognitoBackup/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Cognito User Pool backup and restore scripts
+
+These scripts can be used to create a copy of user entries in a cogntio user pool. They will download user information into a CSV file and then use the CSV file to import users into a new User Pool.
```
Comment (timarney) [inline utils/cognitoBackup/README.md:3]: typo

File: utils/cognitoBackup/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "cognito_user_pool_backup",
+  "version": "0.1.0",
+  "description": "Retrieve and store users form Cognito User Pool",
```
Comment (timarney) [inline utils/cognitoBackup/package.json:4]: `form` should this be `from` ?

File: utils/cognitoBackup/README.md (hunk 1 above)
Comment (bryan-robitaille) [reply utils/cognitoBackup/README.md:3]: Thanks, got to love fat fingers. ;)

File: utils/cognitoBackup/package.json (hunk 2 above)
Comment (bryan-robitaille) [reply utils/cognitoBackup/package.json:4]: 👍

**Review by Moro-Code — APPROVED** [review]
Great work !

**Pull request merged**